en-US/Punctuation.xml

```diff
@@ -647,9 +767,21 @@ PLAY [Gather and display facts for managed nodes] ****************************</
 		  <para>
             The path contains the library qualifier in braces.
           </para>
+          </example>
+		<example><title>Referring to Special Characters</title>
+
+          <para>
+            When a regular user starts a shell, the prompt includes an ending dollar character ($).
+          </para>
+		  <para>
+            Use the pipe character (|) to send the output of one program to the input of another program.
+          </para>
+		  <para>
+            The hyphen after the greater-than sign (>) indicates that a newline character (\n) is not added to the end of the variable.
```

Comment:
According to the table on special character names, the "greater than symbol" does not use a hyphen (even though that would be grammatically correct!).